deploy/minikube/releases-beta.json

```diff
@@ -18,9 +18,9 @@
   {
       "name": "v1.20.0-beta.0",
       "checksums": {
-          "darwin": "",
-          "linux": "",
-          "windows": ""
+          "darwin": "686f8d7c06c93f28543f982ec56a68544ab2ad6c7f70b39ede5174d7bac29651",
```

Comment:
is this change related to this PR?

Reply:
yes, with empty hashes the test for beta versions would fail